modules/RelationalDataLoader.py

```diff
@@ -67,39 +87,81 @@ def raw_connection_string_to_valid_source_connection_string(self, connection_str
-                                 'forcefully refreshed dropping and recreating the destination tables. All others '
-                                 'models would only be refreshed if required as per the state of the source and '
-                                 'destination tables. '
-                                 'Eg \'CompoundPkTest,LargeTableTest\'. '
-                                 'Leave blank or use glob (*) to force full refresh of all models.')
-
-        parser.add_argument('-l', '--log-level',
-                            default='INFO',
-                            type=self.log_level_string_to_int,
-                            nargs='?',
-                            help=f'Set the logging output level. {_LOG_LEVEL_STRINGS}')
+        process_command_parser.add_argument(
+            'destination_connection_string',
+            metavar='destination-connection-string',
+            help='The destination database connection string. Provide in PostgreSQL'
+            ' + Psycopg format. '
+            'Eg: \'postgresql+psycopg2://username:password@host:port/dbname\'')
+
+        process_command_parser.add_argument(
+            'configuration_folder',
+            metavar='configuration-folder',
+            help='Absolute or relative path to the models. '
+            'Eg \'./models\', \'C:/path/to/models\'')
+
+        process_command_parser.add_argument(
+            '-f',
+            '--force-full-refresh-models',
+            nargs='?',
+            const='*',
+            help='Comma separated model names in the configuration folder. '
+            'These models would be forcefully refreshed dropping and recreating the '
+            'destination tables. All others models would only be refreshed if required '
+            'as per the state of the source and destination tables. '
+            'Eg \'CompoundPkTest,LargeTableTest\'. '
+            'Leave blank or use glob (*) to force full refresh of all models.')
+
+        process_command_parser.add_argument(
+            '-l', '--log-level',
+            default='INFO',
+            type=self.log_level_string_to_int,
+            nargs='?',
+            help=f'Set the logging output level. {_LOG_LEVEL_STRINGS}')
+
+        audit_command_parser = subparsers.add_parser('audit',
+                                                     help='provides list of processed models since a given timestamp')
+        audit_command_parser.set_defaults(func=self.execute_audit_command)
+
+        audit_command_parser.add_argument(
+            'destination_connection_string',
+            metavar='destination-connection-string',
+            help='The destination database connection string. Provide in PostgreSQL'
+            ' + Psycopg format. '
+            'Eg: \'postgresql+psycopg2://username:password@host:port/dbname\'')
+
+        audit_command_parser.add_argument(
+            'model_return_type',
+            metavar='model-return-type',
+            choices=['FULL', 'INCR'],
```

Comment:
we can point to a list of constants for this.. then the constants can also be used when reading this input and comparing to decide.